examples/misc/keyword_plotting.py

```diff
@@ -0,0 +1,28 @@
+"""
+======================
+Plotting with keywords
+======================
+
+There are some instances where you have data in a format that lets you
+access particular variables with strings. For example, with
+:class:`numpy.recarray` or :class:`pandas.DataFrame`.
+
+Matplotlib allows you provide such an object with
+the ``data`` keyword argument. If provided, then you may generate plots with
```

Comment:
Well, it's not major, but I just think it looks odd when one line is broken much earlier than the rest; it didn't even break at the end of a sentence.